### PR TITLE
doc updates

### PR DESCRIPTION
**Pull request by lizzerdrix**
* adds info around user_id field
* adds info about the common "sticky" referrer issue on AJAX galleries
* adds info on gallery implementation

### CODE REVIEW

**Comment by lizzerdrix** [comment]
@andremalan @pnts @h55nick ready for review, at your leisure

**Comment by h55nick** [comment]
+1

File: source/best-practices.html.erb [hunk 1]
```diff
@@ -33,6 +33,9 @@ class: best-practices
       <li><strong>Slide Grouping</strong> <em class="recommended">(recommended)</em> - All slides in a slideshow or images in a gallery are grouped together and assigned the canonical URL of the main page of the slideshow or gallery.</li>
   </ol>
   <p>As there are pros and cons to each gallery implementation method, we recommend speaking to one of our Content Analysts to determine the correct approach for your site.</p>
+  <p>Be sure that the SimpleReach code is set to fire on each slide change, with the same canonical URL attributed to the <em>url</em> field for each slide (if 
+    using slide grouping).
```
Comment (pnts) [inline source/best-practices.html.erb]: I'd put the caveat about slide grouping up front, something like, "When using slide grouping, be sure that the SimpleReach code..." this way the dependency for this instruction is read first and the rest can be ignored if they aren't using slide grouping.

File: source/ajax.html.erb [hunk 2]
```diff
@@ -19,4 +19,38 @@ SPR.Reach.collect(__ajax_reach_config);
 
   <p>You can do this as many times as you want for each new AJAX page load.  It is not necessary to use the <code><strong>SPR.Reach.collect()</strong></code> method unless you are making an additional page load calls (i.e. AJAX or PJAX).  The SimpleReach Javascript will attempt initially assume that it is on a static page and send any <code><strong>__reach_config</strong></code> already on the page through the <code><strong>SPR.Reach.collect</strong></code> method on its own.</p>
 
-</article>
\ No newline at end of file
+<div id="ref_url">
+  <h2>Referrer Issues</h2>
+
+  <p>
+    When tracking items in this way, there may be issues with the <em>referrer</em>. The <strong>referrer</strong> is a field that we automatically populate
```
Comment (pnts) [inline source/ajax.html.erb]: Why is referrer italicized in one sentence, and bold in the second?

File: source/ajax.html.erb [hunk 3]
```diff
@@ -19,4 +19,38 @@ SPR.Reach.collect(__ajax_reach_config);
...
+    Sometimes the initial referrer on page load "sticks" across all the slides
+    in a slideshow. A user might arrive to a page via Facebook, and the referrer will be properly attributed to www.facebook.com on the initial
+    slide, but incorrectly attributed as the referrer on every slide in the slideshow as well. If a user navigates to one slide from another, the referrer
+    <em>should</em> be the URL for the previous slide. Since every slide change is collected as a page view, we don't want to inflate social referrals in our tool by incorrectly attributing multiple
+    page views to the same URL.
+  </p>
+
+  <p>
+    If you encounter an issue with "sticky" referrers, <strong>do NOT manually populate the <em>referrer</em> field</strong>. Just add an additional field to your __ajax_reach_config object called
+    <strong>ref_url</strong>. The <em>ref_url</em> field will override the referrer and should be populated with the correct referring URL. This is usually achieved
```
Comment (pnts) [inline source/ajax.html.erb]: Same question as above - why italicized for one ref_url and bold for another?

**Comment by andremalan** [comment]
:+1:

**Comment by pnts** [comment]
Left my notes in the PR. :+1: otherwise.

**Comment by lizzerdrix** [comment]
@pnts updated as per your comments - lemme know if it's good, and I shall DEPLOY!

**Comment by pnts** [comment]
@lizzerdrix :+1: !

**Pull request merged**